app/controllers/import_users_controller.rb

```diff
@@ -70,7 +77,7 @@ def valid_input_file?(imported_users, file) # rubocop:disable all
     unless imported_users.first.keys == accepted_keys
       flash[:error] = 'Unable to import CSV file. Please ensure that the '\
         'first line of the file exactly matches the sample input (username, '\
```

Comment:
Given my [last comment](https://github.com/YaleSTC/reservations/pull/1297/files#r39925293), this is effectively a non-issue since we're going to just strip the first line of all whitespace before parsing the headers.